db/external_sst_file_ingestion_job.cc

```diff
@@ -386,15 +386,25 @@ Status ExternalSstFileIngestionJob::IngestedFilesOverlapWithMemtables(
   return status;
 }
 
-Status ExternalSstFileIngestionJob::AssignLevelForIngestedFile(
-    SuperVersion* sv, IngestedFileInfo* file_to_ingest, bool* overlap_with_db) {
-  *overlap_with_db = false;
+Status ExternalSstFileIngestionJob::AssignLevelAndSeqnoForIngestedFile(
+    SuperVersion* sv, bool force_global_seqno, CompactionStyle compaction_style,
+    IngestedFileInfo* file_to_ingest, SequenceNumber* assigned_seqno) {
+  Status status;
+  *assigned_seqno = 0;
+  const SequenceNumber last_seqno = versions_->LastSequence();
+  if (force_global_seqno) {
+    *assigned_seqno = last_seqno + 1;
+    if (compaction_style == kCompactionStyleUniversal) {
+      file_to_ingest->picked_level = 0;
```

Comment:
why force_global_seqno + useversal style makes us to always go to L0? Because for that combination to ingest on other levels we must use seqno stealing, and forcing seqno prohibits that? That's my understanding.